src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -385,7 +395,10 @@ private void updateProjectItems(bool isEnabled)
 
       public void Handle(SimulationRunStartedEvent eventToHandle)
       {
+         updateSimulationItemsFor(eventToHandle.Simulation);
          _menuBarItemRepository[MenuBarItemIds.Stop].Enabled = true;
+         _menuBarItemRepository[MenuBarItemIds.Run].Enabled = !_simulationState.IsRunning;
+         _menuBarItemRepository[MenuBarItemIds.RunWithSettings].Enabled = !_simulationState.IsRunning;
```

Comment:
isn't this already included in `updateSimulationItemsFor(eventToHandle.Simulation);` ?